.editorconfig

```diff
@@ -0,0 +1,3 @@
+
+[{*.kt,*.kts}]
+indent_size = 2
```

Comment:
you're going to get knackered with this config since you're a mixed java / kotlin project and it will mess up your java imports etc.  Worth not checking it in and using the one from the new plugin instead.

Reply:
although don't do it now otherwise it will create lots of other modifications, will have to be done in a separate PR to upgrade etc.